OpenDreamRuntime/Objects/Types/DreamObjectException.cs

```diff
@@ -0,0 +1,29 @@
+namespace OpenDreamRuntime.Objects.Types;
+
+public sealed class DreamObjectException : DreamObject {
```

Comment:
Apparently C# still wants there to be a constructor since `DreamObject` doesn't have a zero-argument constructor.
```suggestion
public sealed class DreamObjectException(DreamObjectDefinition objectDefinition) : DreamObject(objectDefinition) {
```